utilities/rpp-unittests/HOST_NEW/generatePerformanceLogs.py

```diff
@@ -0,0 +1,116 @@
+import os
+import subprocess
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--case_start", type=str, default="0", help="Testing range starting case # - (0:86)")
+parser.add_argument("--case_end", type=str, default="86", help="Testing range ending case # - (0:86)")
+parser.add_argument('--third_func', type=str, default='0', help='Case # (>0)')
```

Comment:
Rename "third_func" with "unique_func" since that is the name of the parameter that is being used
https://github.com/sampath1117/rpp/blob/sr/test_suite_upgrade_latest/utilities/rpp-unittests/HOST_NEW/testAllScript.sh#L121